### PR TITLE
Introduce combined sigs commit

### PR DESCRIPTION
**Pull request by n-hutton**

### CODE REVIEW

File: x/staking/handler_test.go [hunk 1]
```diff
@@ -162,27 +159,6 @@ func TestDuplicatesMsgCreateValidator(t *testing.T) {
 	assert.Equal(t, Description{}, validator.Description)
 }
 
-func TestInvalidPubKeyTypeMsgCreateValidator(t *testing.T) {
```
Comment (jinmannwong) [inline x/staking/handler_test.go:165]: Why is this test deleted?

File: x/staking/types/msg.go [hunk 2]
```diff
@@ -50,6 +52,10 @@ func NewMsgCreateValidator(
 	description Description, commission CommissionRates, minSelfDelegation sdk.Int,
 ) MsgCreateValidator {
 
+	if _, ok := pubKey.(bls12_381.PubKeyBls); !ok && pubKey != nil {
+		panic(fmt.Sprintf("Attempt to create validator with non bls based public key is invalid!\n"))
```
Comment (jinmannwong) [inline x/staking/types/msg.go:56]: I don't know if this should be a panic. Should we not just reject the msg when the MsgCreateValidator is handled?

File: x/slashing/app_test.go [hunk 3]
```diff
@@ -156,7 +158,7 @@ func TestSlashingMsgs(t *testing.T) {
 	require.Equal(t, sdk.ValAddress(addr1), validator.OperatorAddress)
 	require.Equal(t, sdk.Bonded, validator.Status)
 	require.True(sdk.IntEq(t, bondTokens, validator.BondedTokens()))
-	unjailMsg := MsgUnjail{ValidatorAddr: sdk.ValAddress(validator.ConsPubKey.Address())}
+	unjailMsg := MsgUnjail{ValidatorAddr: validator.OperatorAddress}
```
Comment (jinmannwong) [inline x/slashing/app_test.go:161]: Why did this need to change?

File: x/staking/types/msg.go (hunk 2 above)
Comment (n-hutton) [reply x/staking/types/msg.go:56]: It does not need to be a panic, it is quite useful when making the switch to find out when code has tried to create an incorrect validator. I can remove it if you like?

File: x/slashing/app_test.go (hunk 3 above)
Comment (n-hutton) [reply x/slashing/app_test.go:161]: The address that is signing the unjail message should be the 'operating address' of the validator, which is not related to its consensus public key (now bls).

File: x/staking/types/msg.go (hunk 2 above)
Comment (jinmannwong) [reply x/staking/types/msg.go:56]: No, that's fine. I agree it is useful when switching to keep this.

**Review by jinmannwong — APPROVED** [review]

**Pull request merged**